types/jquery-cropper/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "interface-name": false
```

Comment:
Do not disable the rule here. Fix errors or disable the error at the line where you need to explaining why that is correct

Reply:
unfortunately I'm on vacation now, I'll be back next week, then I can send the updates after 15th of March.
I tried to update it last week but it seemed like the PR was already deleted, that is why I wait until I come back from vacation.